tools/updtset.cmd

```diff
@@ -0,0 +1,235 @@
+
+echo 3. Lets compose %tileset_name%. Be patient it takes some time.
+if /i [!direct_update!] EQU [YES] (
+	rd /q /s "%temp%\cdda_tset_compose"
+	mkdir "%temp%\cdda_tset_compose"
+	xcopy "%the_game_dir%\gfx\%tileset_name%\fallback*.*" "%temp%\cdda_tset_compose" /Y /Q 1>nul
+	xcopy "%the_game_dir%\gfx\%tileset_name%\tileset.txt" "%temp%\cdda_tset_compose" /Y /Q 1>nul
+	if /i [!verbose!] EQU [YES] (echo    - Essential tileset files backed up to [%temp%\cdda_tset_compose].)
+)
+pushd "!path_to_compose!" || goto :deleted
+rd /q /s . 2> NUL
+popd
+:deleted
+py.exe "%script_dir%\compose.py" --use-all "%tileset_fork%\gfx\%tileset_name%" "!path_to_compose!"
+if not errorlevel 1 (
+	if /i [!verbose!] EQU [YES] (echo.)
+
+	echo 4. Now composed tileset will be copied into the game so you can refresh it.
+	if /i [!direct_update!] EQU [NO] (
+		xcopy "!path_to_compose!\*.*" "%the_game_dir%\gfx\%tileset_name%" /Y /Q 1>nul
+		if errorlevel 1 goto stop
+		if /i [!verbose!] EQU [YES] (echo    - All files are available both in)
+		if /i [!verbose!] EQU [YES] (echo      - [!path_to_compose!] and in)
+		if /i [!verbose!] EQU [YES] (echo      - [%the_game_dir%\gfx\%tileset_name%])
+		if /i [!verbose!] EQU [YES] (echo.)
+	) else (
+		xcopy "%temp%\cdda_tset_compose\fallback*.*" "%the_game_dir%\gfx\%tileset_name%" /Y /Q 1>nul
+		xcopy "%temp%\cdda_tset_compose\tileset.txt" "%the_game_dir%\gfx\%tileset_name%" /Y /Q 1>nul
+		if /i [!verbose!] EQU [YES] (echo    - Essential tileset files restored.)
+		if /i [!verbose!] EQU [YES] (echo.)
+	)
+	
+	echo 5. If there any layering info it'll be copied too.
+	if exist "%tileset_fork%\gfx\%tileset_name%\layering.json" (
+		xcopy "%tileset_fork%\gfx\%tileset_name%\layering.json" "%the_game_dir%\gfx\%tileset_name%" /Y /Q 1>nul
+		if errorlevel 1 goto stop
+		if /i [!verbose!] EQU [YES] (echo    - Additional 'layering.json' file copied)
+		if /i [!verbose!] EQU [YES] (echo      - from [%tileset_fork%\gfx\%tileset_name%])
+		if /i [!verbose!] EQU [YES] (echo      -   to [%the_game_dir%\gfx\%tileset_name%])
+	)
+	if /i [!verbose!] EQU [YES] (echo.)
+	
+	echo.
+	echo     All done... Refresh tileset in game. 
+) else (
+	echo ERROR: Something went wrong! && goto stop
+)
+timeout /t 10 >nul
+exit /b 0
+:stop
+echo:
+echo (press any key to close this window)
+pause >nul
+exit /b 1
+
+:: Change log
+:: 1.13
+:: V fixed python launching command. Now it is 'py' everywhere. Sometimes py works, but python launches shortcut that tries to setup Python from Windows store.
+:: V fixed error message, now it appears even in quiet mode.
+:: 1.12
+:: + Added check for pyvips and libvips
+:: + Added installation of pyvips
+:: 1.11
+:: + Added support of paths with spaces
+:: ~ Changed behavior, now verbose is default mode
+:: ~ Changed /v key to /q - quiet mode
+:: 1.10
+:: + Now supports /v /d /s keys
+:: + Also can accept tileset name as argument so it can be provided via shortcut
+:: 1.01
+:: + Added manual reference for python (module+library)
```

Comment:
```suggestion
```
I don't think you need to track changelog in there, it will be tracked by github history

Reply:
that was ages ago )